### DISCUSSION TITLE
Auto Cell

### DISCUSSION THREAD

**Discussion by jagtaprathmesh19** [microsoft/vscode-jupyter#10754] [Questions and Answers] [answered]
Plzz adds cell creation by using the **shift +enter** key because I think it is more flexible to use and most of the Jupyter users also like this feature.

**Comment by IanMatthewHuff** [comment, marked as answer]
@prathmesh-jagtap When you are on the last cell of a notebook document and you press shift+enter that last cell should execute and a new cell is created. Is this what you are referring to? Because that should be how it currently is working.
![ShiftEnter](https://user-images.githubusercontent.com/812783/178157810-e209e32a-828a-49a0-9db9-624c4ecfb02d.gif)

**Reply by jagtaprathmesh19** [reply]
yes, 
but in my vs code, when I press shift + double enter, it works,

**Reply by jagtaprathmesh19** [reply]
Thanks for Answering.

**Reply by IanMatthewHuff** [reply]
That's odd, so for you, you have it hit enter twice? One thing to check is maybe disable extensions other than Jupyter + Python and try it. Maybe there is some other extension trying to handle that keystroke as well and causing an issue.

**Reply by jagtaprathmesh19** [reply]
Yes, it may be
